packages/st2/debian/control

```diff
@@ -15,7 +15,7 @@ Vcs-Browser: https://github.com/stackstorm/st2
 Package: st2
 Architecture: any
 Pre-Depends: dpkg (>= 1.16.16), ${pre:Depends}, ${misc:Pre-Depends}, adduser
-Depends: ${shlibs:Depends}, ${misc:Depends}, sudo, adduser, ${Depends}, libssl-dev, libffi-dev, git, libpam0g, openssh-server, openssh-client, bash, netbase
+Depends: ${shlibs:Depends}, ${misc:Depends}, sudo, adduser, ${Depends}, libssl-dev, libffi-dev, git, libpam0g, openssh-server, openssh-client, bash, netbase, crudini
```

Comment:
Looks like `crudini` was needed in a earlier version of this PR when generators were bash-based.
In a new version of python I don't see where `crudini` is used, looks like could be removed from the dependencies.

Reply:
oh. good catch.